src/main/webapp/js/userMapData.json

```diff
@@ -644,19 +674,23 @@
     "Mill Chase Academy",
     "Newcastle University",
     "Quarrydale Academy",
+    "Queen Margaret University",
     "Robert Gordon University",
     "SECAmb TtP",
     "Southampton Solent University",
+    "Southampton University",
```

Comment:
Check if this is the same as either `Southampton Solent University` or `University of Southampton`.